routes/chorals.js

```diff
@@ -182,13 +189,21 @@ router.get('/:choralId', function(req, res, next) {
     return Promise.all(Object.keys(childData).map((choralId) => {
       return new Promise((resolve, reject) => {
         client.hgetall(choralId, (err, data) => {
-          if(err) {
+          if(err || data == null) {
             logHelper.createLog(
               "error",
               'Choral data has not yet been published to redis: ' + err
               , ["chorals", "get"]
             );
-            reject(err);
+            dataObj = {
+              choralInfo: {
+                sampleRate: choral.sampleRate,
```

Comment:
I'm getting `choral is not defined` here. It looks like  `choral` is never available in this scope AFAICT.